src/rustdoc-internals/htmldocck.md

```diff
@@ -0,0 +1,101 @@
+# `rustdoc` tests
+
+This page is specifically about the `rustdoc` test suite, for other test suites used for testing rustdoc, see [Rustdoc § Tests](../rustdoc.md#tests).
+
+`htmldocck.py` is a custom checker script that uses [XPath] to verify the HTML output of rustdoc.
+
+[XPath]: https://en.wikipedia.org/wiki/XPath
+
+## Directives
+Directives to htmldocck are similar to those given to `compiletest` in that they take the form of `//@` comments.
```

Comment:
This paragraph doesn't make it clear that all regular compiletest directives are supported, too.

Reply:
(Not by HtmlDocCk directly but by compiletest since the rustdoc/ test suite is both checked by compiletest *and* HtmlDocCk by virtue of compiletest invoking/driving HtmlDocCk)